lib/ruleHelper.js

```diff
@@ -128,6 +128,7 @@ RuleHelper.prototype = {
             methodName = node.property.name;
             objectName = node.object.name || this.context.getSource(node.object);
             break;
+        case "CallExpression":
```

Comment:
I *think* you could fix #176 if you also added "ConditionalExpression" to return an empty method name

Reply:
@mozfreddyb I'm guessing you were meant to refer to #171 and/or #174, is that right?

Anyway I also think that you are right, I've been able to trigger errors like the ones reported in #171 and #174 using a code snippet like this:
```
a.innerHTML = (cond ? maybe_safe : or_evil)();
```

and adding "ConditionalExpression" along with "CallExpression" and "ArrowFunctionExpression" prevented it and looks a readonable addition.

I added an additional patch to this pull request.

Reply:
uh, yes. wrong bug id. thanks!